src/js/common/schemaform/GetPensionOrBurialFormHelp.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function GetFormHelp() {
+  return (
+    <div>
+      <p className="talk">For benefit related questions please call:</p>
+      <p className="phone-number">
+        <a href="tel:+1-800-827-1000">800-827-1000</a><br/>
+        Monday - Friday, 8:00 a.m. - 9:00 p.m. ET<br/>
+        <p className="talk">For Telecommunications Relay Service (TRS): dial <a href="tel:711">711</a></p>
```

Comment:
Does this make sense as a tel link? Is it an extension? Is it what we punch in after we call to get to the telecommunications relay service (whatever that is)?